src/components/calendar-graph/calendar-graph.scss

```diff
@@ -0,0 +1,155 @@
+@import '@/assets/colors';
+@import '@/assets/styles';
+
+$month-label-hight: 20px;
+$vertical-margin: 10px;
+
+@mixin hidable-label {
```

Comment:
spelling if you care at all XD

Reply:
my spell checker did me dirty:sob::sob:. It said `hidable` was good but `hideable` wasn't recognized.

Reply:
webster dictionary has me questioning what is right... but for some reason `hidable` looks so wrong 😆